t04_refactored.py

```diff
@@ -53,11 +53,12 @@ def end_story(user):
     :param user: the user's name
     :return: None
     """
-    print("Congratulations, " + user + ", you have made it to the end of this... strange... adventure. I hope you feel accomplished.")
```

Comment:
Changes should not have been made outside of your function. Please revert these.